### PR TITLE
Thuang fix alert

### PR DESCRIPTION
**Pull request by tihuan**
This PR fixes Alert component using the component library's default theme instead of the product's default theme 😆 

PTAL thank you!!

### CODE REVIEW

File: src/core/Alert/style.ts [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
-  margin: ${defaultTheme.spacing(5)}px 0;
-  border-radius: ${defaultTheme.spacing(1)};
-  color: ${defaultTheme.palette.text.primary};
-  padding: ${defaultTheme.spacing(6)}px ${defaultTheme.spacing(6)}px
-    ${defaultTheme.spacing(6)}px 9px;
+  ${(props) => {
+    const colors = getColors(props);
+    const spacings = getSpacings(props);
+    const shadows = getShadows(props);
+    const { severity = "primary" } = props;
```
Comment (phoenixAja) [inline src/core/Alert/style.ts:11]: cool i didn't realize that you could unpack an object like this!

**Review by phoenixAja — APPROVED** [review]
TY for addressing this so quickly timmy!!

File: src/core/Alert/style.ts (hunk 1 above)
Comment (tihuan) [reply src/core/Alert/style.ts:11]: Yaas it works for arrays too!

`const [a, b] = [1, 2]`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

**Comment by tihuan** [comment]
Thanks for the quick review, @phoenixAja !! Merging!

**Pull request merged**